tutorials/ios-chatbot/part-2/index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: How to build a conversational app using Cloud Machine Learning APIs (Part 2 of 3)
+description: How to build a conversational app using Cloud Machine Learning APIs (Part 2 of 3).
+author: Chang Luo
+tags: Cloud Functions, Dialogflow, API.AI, Webhooks, Localization, Chatbot, Machine Learning API, Transation, Vision, Speech
+date_published: 2017-01-02
+---
+
+In [part 1](../index.md) of this blog post, we gave you an overview of what a conversational tour guide iOS app might look like built on Cloud Machine Learning APIs and API.AI. We also demonstrated how to create API.AI intents and contexts. In part 2, we'll discuss an advanced API.AI topic — webhook with Cloud Functions. We'll also show you how to use Cloud Machine Learning APIs (Vision, Speech and Translation) and how to support a second language.
+
+### Webhooks via Cloud Functions
+
+In API.AI, [Webhook](https://docs.api.ai/docs/webhook) integrations allow you to pass information from a matched intent into a web service and get a result from it. Read on to learn how to request parade info from Cloud Functions.
```

Comment:
Throughout the doc, you have a mix of both uppercase and lowercase for webhook. Doing a quick search shows that most people use lowercase.